tests/python/gaia-ui-tests/gaiatest/apps/email/regions/setup.py

```diff
@@ -182,7 +183,10 @@ def tap_next(self):
                 *self._account_prefs_next_locator))))
 
     def check_for_emails_interval(self, value):
-        self.marionette.execute_script('document.querySelector("[data-l10n-id = settings-check-every-5min]").value = "%s";' % value)
+        self.marionette.execute_script("""
+            SpecialPowers.setIntPref('dom.requestSync.minInterval', 1);
```

Comment:
A comment explaining why we change the data_layer in a function which manipulates the UI would be helpful for future readers.